cobra2snooty.go

```diff
@@ -119,9 +122,40 @@ func GenDocs(cmd *cobra.Command, w io.Writer) error {
 	}
 	printOptions(buf, cmd)
 
-	if len(cmd.Example) > 0 {
+	if len(s) > 0 && len(s) < 2 {
+		if len(cmd.Example) > 0 {
+			buf.WriteString(examplesHeader)
+			buf.WriteString(fmt.Sprintf("\n%s\n", indentString(cmd.Example, " ")))
+		}
+	}
```

Comment:
```suggestion
	if len(s) == 1) {
		buf.WriteString(examplesHeader)
		buf.WriteString(fmt.Sprintf("\n%s\n", indentString(cmd.Example, " ")))
	}
```

Reply:
I changed this block, but we do still need the if len(cmd.Example) > 0 it seems because otherwise it prints an empty code block for the ones that don't have examples. I made it part of the same if statement.